### ISSUE TITLE
TypeError: 'str' object cannot be interpreted as an integer

### ISSUE THREAD

**Issue by arsaboo** [dmdhrumilmistry/MusicAPy#19]
I am using MusicAPy in a beets plugin and I am getting this error when searching for an album:
```bash
Traceback (most recent call last):
  File "/home/arsaboo/.local/bin/beet", line 8, in <module>
    sys.exit(main())
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/__init__.py", line 1301, in main
    _raw_main(args)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/__init__.py", line 1288, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/commands.py", line 1034, in import_func
    import_files(lib, paths, query)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/commands.py", line 974, in import_files
    session.run()
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/importer.py", line 345, in run
    pl.run_parallel(QUEUE_SIZE)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/util/pipeline.py", line 446, in run_parallel
    raise exc_info[1].with_traceback(exc_info[2])
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/util/pipeline.py", line 311, in run
    out = self.coro.send(msg)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/util/pipeline.py", line 170, in coro
    task = func(*(args + (task,)))
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/importer.py", line 1460, in user_query
    task.choose_match(session)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/importer.py", line 896, in choose_match
    choice = session.choose_match(self)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/commands.py", line 780, in choose_match
    post_choice = choice.callback(self, task)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/ui/commands.py", line 710, in manual_id
    _, _, prop = autotag.tag_album(
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/autotag/match.py", line 473, in tag_album
    for album_info_for_id in hooks.albums_for_id(search_id):
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/autotag/hooks.py", line 652, in albums_for_id
    for a in plugins.album_for_id(album_id):
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beets/plugins.py", line 399, in album_for_id
    album = plugin.album_for_id(album_id)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/beetsplug/jiosaavn.py", line 220, in album_for_id
    album_details = self.jiosaavn.get_album_details(id)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/musicapy/saavn_api/services.py", line 269, in get_album_details
    song_details = SongService.get_song_details(song_identifier, use_v4=False)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/musicapy/saavn_api/services.py", line 87, in get_song_details
    song_details = get_data(api_type, param, use_v4)
  File "/home/arsaboo/.local/lib/python3.10/site-packages/musicapy/saavn_api/endpoint.py", line 41, in get_data
    data.pop('modules')
TypeError: 'str' object cannot be interpreted as an integer
```

You can reproduce the error using the following code:
```python:
from musicapy.saavn_api.api import SaavnAPI

# create API obj
api = SaavnAPI()

saavn_song_link = 'https://www.jiosaavn.com/album/high-jack/v-,5W1NUOFU_'
identifier = api.create_identifier(saavn_song_link, 'album')
api.get_album_details(identifier)
```

**Comment by dmdhrumilmistry** [comment]
It seems like, JioSaavn might have updated their API responses. Would need to reverse engineer their APIs again to fix this issue.

**Comment by dmdhrumilmistry** [comment]
Fix PR #20
